fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/CharacterService.java

```diff
@@ -21,7 +21,7 @@ public class CharacterService {
     public List<Character.CharacterCreateDto> getAllCharacters(int bookId) {
         if (bookId == 1) {
             return List.of(new Character.CharacterCreateDto(new PlayerSession.PlayerDto("Liana The Swift", SectionId.book1(1), Collections.emptySet(),RankEnum.OUTCAST, ProfessionEnum.WAYFARER, 9,
-                            new Character.BaseStatsDto(2, 5, 2, 3, 6, 4), List.of("spear", "leather jerkin (Defence +1)", "map"),new ShardSystem(16),Map.of(),Collections.emptySet()),
+                            new Character.BaseStatsDto(2, 5, 2, 3, 6, 4), List.of("spear", "leather jerkin (Defence +1)", "map"),new ShardSystem(16),Map.of(),Collections.emptySet(),Collections.emptySet()),
```

Comment:
we should start using a Builder pattern for `PlayerDto`, like https://projectlombok.org/features/Builder .
Otherwise those constructors will get out of hands...